client/error.go

```diff
@@ -0,0 +1,11 @@
+package client
+
+import "github.com/juju/errgo"
+
+// attempt to create a client without endpoint
+var endpointNotSpecifiedError = errgo.New("no endpoint has been specified")
```

Comment:
Godoc usually starts with the variable/member/function name.

Reply:
I do that for exported variables (uppercase first letter), but not for non exported ones. IMHO the dublication makes the comment harder to read directly in the code.

Reply:
> IMHO the duplication makes the comment harder to read directly in the code.

I understand it doesn't fit your personal taste. This is established convention, so let's conform.